lib/internal/test_runner/test.js

```diff
@@ -263,6 +263,16 @@ class TestContext {
           return ReflectApply(method, this, args);
         };
       });
+
+      // assert.ok needs it's own function, as it checks the stack
+      // trace to provide more descriptive errors.
+      function ok(...args) {
+        if (plan !== null) {
+          plan.actual++;
+        }
+        innerOk(ok, args.length, ...args);
```

Comment:
Why not pass `args` directky?
```suggestion
        innerOk(ok, args);
```

Reply:
IMO we should change to what I suggested, I assume the assert function is like that only because it's exposed and we can't make breaking change to it without a deprecation cycle.